packages/core/src/analytics/integrations/Omnitracking/definitions.js

```diff
@@ -523,6 +524,85 @@ export const trackEventsMapper = {
     wishlistId: data.properties?.wishlistId,
     lineItems: getProductLineItems(data),
   }),
+  [eventTypes.PRODUCT_UPDATED]: data => {
+    const eventList = [];
+    const properties = data.properties;
+
+    if (properties.colour && properties.oldColour !== properties.colour) {
+      // color changed event
+      const additionalParameters = {};
+      if (properties?.oldColourId && properties?.colourId) {
+        additionalParameters[
+          'colourList'
+        ] = `${properties?.oldColourId}, ${properties?.colourId}`;
+      } else {
+        logger.warn(
```

Comment:
Question: If `oldColourId` and `colourId` are missing from the tracked event, should we track the event anyway? I do not remember if `colourList` parameter was optional or not. If it is required, then I would change the outer if to check for the id properties instead of the description ones.

Reply:
when don't have old and new color ids, then we can't understand if some color was changed into another.

Reply:
Ok, understood. Thanks!